pkgs/applications/editors/emacs/elisp-packages/manual-packages/lsp-bridge/default.nix

```diff
@@ -0,0 +1,66 @@
+{ lib
+, trivialBuild
+, fetchFromGitHub
+, python3Packages
+, python3
+, posframe
+, markdown-mode
+, yasnippet
+, org
+, which-key
+, makeWrapper
+}:
+
+let
+  rev = "7dfeeb640d14697755e2ac7997af0ec6c413197f";
+  python = python3.withPackages (ps: with ps; [ epc orjson sexpdata six ]);
+in trivialBuild {
+  pname = "lsp-bridge";
+  version = "20230104";
```

Comment:
```suggestion
  version = "unstable-2023-01-04";
```

Reply:
If using `trivialBuild`, `unstable-YYYY-MM-DD` is probably the best choice to fit in with the rest of nixpkgs, but if we want to generate ELPAs we need to fit the format expected by Emacs: https://github.com/emacs-mirror/emacs/blob/master/lisp/subr.el#L6670-L6751 (which is typically just the date shoehorned to fit into a MAJOR.MINOR version)